page-conf.php

```diff
@@ -3,7 +3,7 @@
 		<img class="conf-img" src="<?php echo get_template_directory_uri(); ?>/assets/img/conference/conf-logo.svg" alt="logo" />
 		<h2 class="conf-location">August 28 - Sept 3 2015, Berlin</h2>
 		<a href="https://conference.owncloud.org/conference/oCC2015/register" style="font-weight: 300" class="btn btn-lg btn-primary">Register <i class="icon-check"></i></a>
-		<h3>Bringing ownCloud Contributors from around the world together for a week of coding, design, discussion, talks & fun.</h3><br />
+		<h3>The annual ownCloud Contributor Conference brings ownCloud Contributors from around the world together for a week of coding, design, discussion, talks & fun.</h3><br />
```

Comment:
This is a bit much duplication though ;)

> The annual ownCloud Contributor Conference brings ownCloud Contributors

Before was better, it needs to be catchy.